Projectoid.js

```diff
@@ -0,0 +1,313 @@
+//The Global Variables
+let libPrefix = "projectoid";
+let API_URL = "https://api.projectoid.site/v1";
+
+// Function to convert JSON to URL query string
+function jTQS(jsonData) {
+  let queryString = Object.keys(jsonData)
+    .map(function (key) {
+      return encodeURIComponent(key) + "=" + encodeURIComponent(jsonData[key]);
+    })
+    .join("&");
+  return queryString;
+}
+
+/**
+ * apiCall() - Makes an API call to Projectoid.
+ *
+ * @since 1.0.0
+ *
+ * @param {object} options - The API call options.
+ */
+function apiCall(options) {
+  if (!options) {
+    throw libPrefix + ": apiCall: options not found";
```

Comment:
options not passed ?

Reply:
what do you means options are not passed?

This function can be used directly in bot too, that is why we need to check if the options are passed or not.